### DISCUSSION TITLE
Memory parameters

### DISCUSSION THREAD

**Discussion by seanyoung** [bengtmartensson/IrpTransmogrifier#243] [General]
A parameter spec can have the `@` attribute, but I can't figure out what this means semantically. I've looked at the code and search the documentation, and still I don't understand.

What did I miss?

Thank you very much.

**Comment by bengtmartensson** [comment]
It is described [here](http://www.harctoolbox.org/IrpTransmogrifier.html#Persistency+of+variables). For an example, see ProtocolNGTest.java, testToIrSignalRc5().

**Reply by seanyoung** [reply]
Of course, not sure why I missed that. Thank you very much!
